taskflow/tasks.py

```diff
@@ -192,6 +193,9 @@ def __init__(self, func=None, args=None, max_runs=None, needs_prev_result=True,
         super().__init__(max_runs=max_runs, needs_prev_result=needs_prev_result, name=name)
         self._func = func
         self._args = args or []
+        self._date_updated = None
```

Comment:
```suggestion
        self._execution_end_time = None
```
Sounds more consistent to me 🤔